third_party/move/tools/move-coverage/src/coverage_map.rs

```diff
@@ -277,7 +277,7 @@ impl TraceMap {
         for line in BufReader::new(file).lines() {
             let line = line.unwrap();
             let mut splits = line.split(',');
-            let exec_id = splits.next().unwrap();
+            let exec_id = "exec_id";
```

Comment:
Same as above: note that this is a dummy (or better yet, call it `"dummy_exec_id"` in both places).